### PR TITLE
fix(cvsb-19156): updated error message

### PR DESCRIPTION
**Pull request by waringr**
## Ticket title
Update error messages as per testing

## Checklist
- [x] Code has been tested manually
- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by sdh100shaun — APPROVED** [review]

File: src/domain/getCertificate.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import CertificateDetails from '../interfaces/CertificateDetails';
 import getObjectFromS3 from '../infrastructure/s3/s3Service';
 import validate from '../utils/validationService';
 import NoBodyError from '../errors/NoBodyError';
-import CertificateNumberError from '../errors/CertificateNumberError';
+import TestNumberError from '../errors/TestNumberError';
```
Comment (JonLazariniDD) [inline src/domain/getCertificate.ts:7]: Something to discuss  as devs but I am thinking that we could be using barrel files with a named import where it is consumed so we avoid all these individual imports. I suspect that the list could grow. What do you think please?

Comment (waringr) [reply src/domain/getCertificate.ts:7]: By "barrel files" do you mean having multiple error classes in the same file and then named exports? Cause I'm fine with that.

Comment (waringr) [reply src/domain/getCertificate.ts:7]: was that a request for me to do that now, or is it alright as is?

**Review by SanjeetBJSS — APPROVED** [review]
Low impact changes, happy with all that.

**Pull request merged**